Source/Core/Core/PowerPC/Interpreter/Interpreter_FPUtils.h

```diff
@@ -268,7 +268,7 @@ inline FPResult NI_madd(UReg_FPSCR* fpscr, double a, double c, double b)
     return result;
   }
 
-  result.value += b;
+  result.value = std::fma(a, c, b);
```

Comment:
Is there still a point in checking if `a*c` is a NaN? Especially as the `isnan(b)` handling is in there as well?